packages/frontend/tests/integration/components/pod/chat/chat-entry/emoji-test.ts

```diff
@@ -0,0 +1,88 @@
+import { module, test } from 'qunit';
+import { render, fillIn } from '@ember/test-helpers';
+import { setupRenderingTest } from 'ember-qunit';
+import { hbs } from 'ember-cli-htmlbars';
+import { getStore } from 'emberclear/tests/helpers';
+
+module('Integration | Component | chat-entry', function(hooks) {
```

Comment:
there is one more test missing from this:
 - where we enter text, emojis get replaced, and then we enter more text, and more emojis get replaced.

Reply:
Done